partner_create_by_vat/views/res_partner_view.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="view_partner_form" model="ir.ui.view">
+            <field name="name">res.partner.vat.inherit</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="base_vat.view_partner_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//sheet" position="before">
+           		    <header>
+ 				        <button name="button_get_partner_data" attrs="{'invisible': [('name', '&lt;&gt;', ' '),('vat', '=', False),('is_company', '=', False )]}" colspan="2" string="Get Partner Data" context="{'active_id': active_id}" type="object" icon="gtk-execute"/>
+                	</header>  
+           	    </xpath>
+                <label for="vat" position="replace"/>
+                <div name="vat_info" position="replace">
```

Comment:
I don't quite like the base_vat div, since the label is on the div, so you have vat, the button and vat_subjected field in the same labelling, and you really don't know what the tickbox (vat_subjected) represents...